tools/memleak.py

```diff
@@ -119,6 +121,12 @@ def run_command_get_pid(command):
 min_size = args.min_size
 max_size = args.max_size
 obj = args.obj
+sort_key = args.sort
+
```

Comment:
How about defining and using this dictionary object?
sort_keys = {"size": lambda a: a.size,
 "count: lambda a: a.count}

Reply:
I changed the code as you suggested. Thanks!